docs/conf.py

```diff
@@ -84,8 +84,8 @@
 autodoc_default_options = {
     "exclude-members": (
         "__repr__, __weakref__, __metaclass__, __init__, __format__,  __new__, __str__, __dir__,"
-        "model_config, model_fields, model_post_init, model_computed_fields,"
-        "__ape_extra_attributes__,"
+        "model_config, model_dump, model_fields, model_post_init, model_computed_fields,"
+        "__ape_extra_attributes__, __get_item__, __contains__, __hash__,"
```

Comment:
`__contains__` and `__getitem__` may also occasionally be overriden and custom

Reply:
If there are some that are black, for any dunderized methods, you can ignore them per construct directly, just ends up being more verbose, but that is ok

Reply:
If that's the case then just the `model_dump` should be excluded since that's a pydantic model dump and shouldn't be overridden by anything that we do.

Reply:
correct, model_dump should be excluded
but there are maybe some other models where we should exclude `__iter__` and `__contains__` etc. we just cannot do it globally.